test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +37,27 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
As discussed in the tutorial, these tests can be separated into multiple more specific methods. Refer to https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html for our standard of naming the methods.